src/app/_components/Specialists/SpecialistList.js

```diff
@@ -1,13 +1,22 @@
 'use client';
 
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import { CardOrganization, CardSpecialist } from '@components/CardSpecialist';
 import { useListEntries } from '@hooks';
-import { CircularProgress } from '@mui/material';
 import { useSearchParams } from 'next/navigation';
 import _ from 'lodash';
-import { NoInfoToShow } from '../NoInfoToShow';
+import { Slider } from '@components/Slider';
+import { SwiperSlide } from 'swiper/react';
```

Comment:
You can export it from @components/Slider

Reply:
My bad. Thanks! Fixed it